### DISCUSSION TITLE
Change IP address

### DISCUSSION THREAD

**Discussion by arjoshi89** [Security-Onion-Solutions/securityonion#1755] [Unsupported Versions] [answered]
Distributed Deployment with 1 Manager and 2 Sensors. Version:2.3

How can I change IP address for Manager and Sensor? Change Settings from DHCP to Static also.

Is it possible to change IP or will it be available in new release?

**Comment by stijnpieters** [comment, marked as answer]
according to the [documentation](https://docs.securityonion.net/en/2.3/ip.html?highlight=changing%20ip#changing-ip-addresses), this is not yet supported, but will be in the future.
